addon/components/bs-navbar.js

```diff
@@ -295,8 +286,12 @@ export default class Navbar extends Component {
    * @default 'lg'
    * @public
    */
-  @defaultValue
-  toggleBreakpoint = 'lg';
+  get toggleBreakpoint() {
+    if (this.args.toggleBreakpoint === undefined) {
+      return 'lg';
+    }
+    return this.args.toggleBreakpoint;
```

Comment:
Can we simplify to the following or do I miss something?

```suggestion
    return this.args.toggleBreakpoint ?? 'lg';
```